src/components/comments/Comment.jsx

```diff
@@ -0,0 +1,145 @@
+import {Comment, Segment} from "semantic-ui-react";
+import React, {Component} from "react";
+import PropTypes from 'prop-types';
+import {Link} from "react-router-dom";
+import avatar from "../../assets/images/guy.jpg";
+import CommentReplyListContainer
+    from "../../containers/Comments/CommentReplyListContainer";
+import CommentBodyContainer
+    from "../../containers/Comments/CommentBodyContainer";
+import ConfirmationPopup from "./ConfirmationPopup";
+
+class SingleComment extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            repliesVisible: false,
+            editing: false
+        };
+    }
+
+    toggleVisibility(showComments = false) {
+        const {repliesVisible} = this.state;
+        this.setState({
+            repliesVisible: !repliesVisible || showComments,
+            editing: false
+        });
+    }
+
+    render() {
+        const {comment, editComment, deleteComment, deleteLoading, type, loading, status, loggedInUser} = this.props;
+        let displayImage = comment.author.image ? comment.author.image : avatar;
+        let author = comment.author.first_name + " " + comment.author.last_name;
```

Comment:
This line is too long. It has `90` characters.
`comment.author` is duplicated. You can extra `image`, `first_name`,
`last_name`, `username` with `destructuring`. Since the backend has snake style, we can convert them to camel case when destructuring. e.g
`const {a_b: aB} = someObject`